.github/workflows/ci.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
     # Will need to bump this version in the future when there are breaking changes.
     # NB: update tools/integrity.bzl at the same time!
     env:
-      RULES_PY_RELEASE_VERSION: 1.1.0
+      RULES_PY_RELEASE_VERSION: 1.2.0
```

Comment:
The update to the patch does not longer work with `1.1.0`. But `1.2.0` is broken because of the patch. Not sure if we can get the `test-release` job green.